packages/app/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @viron/app
 
+## 2.13.0
```

Comment:
execute `npm run changelog:version` command and this change is automatically generated.